.jenkins/pipelines/standalone/azure-sdk-tests.Jenkinsfile

```diff
@@ -12,6 +12,8 @@ pipeline {
         string(name: "PULL_REQUEST_ID", defaultValue: "", description: "If you are building a pull request, enter the pull request ID number here. (ex. 789)")
         choice(name: "TEST_CONFIG", choices: ['None', 'Nightly', 'Code Coverage'], description: "Test configuration to execute")
         string(name: "COMMIT_SYNC", defaultValue: "", description: "optional - used to sync outputs of parallel jobs")
+        string(name: "PACKAGE_NAME", description: "optional - release package to install (do not include extension)")
```

Comment:
Remove "optional" from param description

Reply:
`PACKAGE_NAME` is optional for standalone pipelines

Reply:
Ok, could you make sure all optional parameters have a `defaultValue` please? Either set to blank or null. Thanks!